src/Tests/AzureClientTests.cs

```diff
@@ -55,21 +92,177 @@ public void TestAzureExecutionTarget()
 
             targetId = "ionq.targetId";
             executionTarget = AzureExecutionTarget.Create(targetId);
-            Assert.IsNotNull(executionTarget);
```

Comment:
Is this assert not required to validate if Create() returned a valid object?

Reply:
Correct, it's redundant here, because the following asserts will fail if `executionTarget` is null.